probe-rs-cli-util/Cargo.toml

```diff
@@ -17,6 +17,7 @@ default = ["sentry"]
 
 [dependencies]
 structopt = "0.3.16"
+thiserror = "1.0"
 anyhow = "1.0"
```

Comment:
I think this can be removed now.

Reply:
I made this an optional dependency as it is still required for sentry. At its current state we could also talk about removing sentry entirely ... but if we do that I'd do it in another PR.